README.md

```diff
@@ -977,6 +977,7 @@ Table of Contents
   * [YepCode](https://yepcode.io) - All-in-one platform to connect APIs and services in a serverless environment. It brings all the agility and benefits of NoCode tools, but with all the power of use programming languages. Free tier includes [1.000 yeps](https://yepcode.io/pricing/).
   * [mogenius](https://mogenius.com/) - A code-to-cloud platform to easily deploy any service, from static website to advanced microservice architectures. mogenius comes with fully managed hyper-scaling cloud resources, Kubernetes, CI/CD and security from Cloudflare. Free tier includes 0.5 CPU core, 1 GB RAM, 5 GB traffic, 2 GB SSD persistent storage.
   * [WayScript](https://www.wayscript.com)- WayScript is an [internal developer platform (IDP)](https://blog.wayscript.com/what-is-an-internal-developer-platform-idp/) that allows software engineers to self-serve secure, cloud based (Docker + k8s) developer environments at any time.  Remove the need to get resources provisioned on a one off basis. Quickly build tools, apps, APIs, dashboards, schedule cron tasks and more. 100 hours of [free runtime per month](https://www.wayscript.com/pricing).
+  * [Napptive](https://napptive.com/) - A managed cloud-native application platform that simplifies the usage of Kubernetes by using the Open Model Application (OAM) abstraction. Free tier includes 1 CPU core, 4GB RAM, and 1GB ephemeral storage. Environments are reset every 24 hours.
```

Comment:
I am not sure a 24 hour reset service would qualify for this list, quoting the readme:

> If the free tier is time-bucketed it has to be for at least a year.

Reply:
Hi! Thanks for the feedback, I would try to clarify the point. The free tier does not have a limitation with regards to time. You can use it as much as you like, but the environments in which you deploy applications are reseted every 24 hours. Users can relaunch their apps as many times as they want. We believe this could be useful for both users wanting to try out a Kubernetes/OAM service, but also for teaching scenarios where applications are not likely to run more than 24 hours.

Reply:
Ok, thanks for the consideration and the feedback. I will resubmit the link if the free tier conditions change in the future.